tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/Domain/ProvisioningTypeTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentAssertions;
+using NHSD.BuyingCatalogue.Infrastructure;
+using NHSD.BuyingCatalogue.Solutions.Application.Domain.Pricing;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.UnitTests.Solutions.Domain
+{
+    [TestFixture]
+	public class ProvisioningTypeTests
+	{
+        [Test]
+        public void EnsureProvioningTypeNamingMatchesOrdapi()
```

Comment:
I think this could be improved by using a TestCase attribute but this will work fine as it is. Usually find it easier to read when there is only one assert statement.